_studio/mfx_lib/decode/av1/src/mfx_av1_dec_decode.cpp

```diff
@@ -450,6 +450,35 @@ mfxStatus VideoDECODEAV1::Query(VideoCORE* core, mfxVideoParam* in, mfxVideoPara
 #endif
     }
 
+    if (in)
+    {
+        switch(in->mfx.CodecLevel)
```

Comment:
can we wrap checking to separate helper function? Just to not expand code of the Query

Reply:
Sure. I have changed it in the latest commit.